manifests/quick-start/base/executor/kubelet/executor-role.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: executor
+  annotations:
+    workflows.argoproj.io/description: |
+      Recomended minimum permissions for `kubelet` executor.
+
+      This executor is superceded be the `emmisary` executor, so we do not recommend you use it anymore.
```

```suggestion
      This executor is superseded by the `emmisary` executor, so we do not recommend you use it anymore.
```

Same in other places